infra/gcp/terraform/k8s-infra-oci-proxy-prod/terraform.tfvars

```diff
@@ -22,263 +22,251 @@ cloud_run_config = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://eu.gcr.io"
+        value = "https://europe-west2-docker.pkg.dev"
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
-      }
-    ]
-  }
-  europe-west3 = {
-    environment_variables = [
-      {
-        name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://eu.gcr.io"
-      },
-      {
-        name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
       }
     ]
   }
   europe-west4 = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://eu.gcr.io"
+        value = "https://europe-west4-docker.pkg.dev"
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
       }
     ]
   }
   europe-west8 = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://eu.gcr.io"
+        value = "https://europe-west8-docker.pkg.dev"
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
       }
     ]
   }
   europe-west9 = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://eu.gcr.io"
+        value = "https://europe-west9-docker.pkg.dev"
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
       }
     ]
   }
   southamerica-west1 = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://us.gcr.io"
+        value = "https://southamerica-west1-docker.pkg.dev"
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
       }
     ]
   }
   us-central1 = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://us.gcr.io"
+        value = "https://us-central1-docker.pkg.dev"
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
       }
     ]
   }
   us-east1 = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://us.gcr.io"
+        value = "https://us-east1-docker.pkg.dev"
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
       }
     ]
   }
   us-east4 = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://us.gcr.io"
+        value = "https://us-east4-docker.pkg.dev"
```

Comment:
```
$ crane pull us-east4-docker.pkg.dev/k8s-artifacts-prod/images/pause:3.2 /dev/null && echo $?
0
```
(AKA, this region does exist and is functioning, xref: https://github.com/kubernetes/registry.k8s.io/issues/116)